lib/hello/accounts.ex

```diff
@@ -0,0 +1,86 @@
+defmodule Hello.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Hello.Repo
+
+  alias Hello.Accounts.{User, Credential}
+
+
+  def list_users do
+    User
+    |> Repo.all()
+    |> Repo.preload(:credential)
+  end
+
+  def get_user!(id) do
+    User
+    |> Repo.get!(id)
+    |> Repo.preload(:credential)
```

Comment:
@JuanMaGiraldo what a preload does?

Reply:
`preload` preloads the associations, in this case, I'm preloading the schema Credential to use it, if I don't preload it, so, I'm just going to get the association as in the image:

![image](https://user-images.githubusercontent.com/33075442/125847980-16a271f1-b61b-4fb4-8edb-1660f36baf0e.png)